src/hvm.c

```diff
@@ -300,11 +305,37 @@ u32 global_sum(u32 x) {
   return sum;
 }
 
+#ifdef _WIN32
+static int64_t _win_time_offset = 0;
+static LARGE_INTEGER _win_time_freq, _win_perf_offset;
+
+static inline void _win_start_timer() {
+  QueryPerformanceFrequency(&_win_time_freq);
```

Comment:
Why not just use the performance counter directly? We dont care about system time, this is just a timer.
```
#ifdef _WIN32
static inline u64 time64() {
  LARGE_INTEGER freq;
  LARGE_INTEGER counter;
  QueryPerformanceFrequency(&freq);
  QueryPerformanceCounter(&counter);
  return (u64)((counter.QuadPart * 1000000000) / freq.QuadPart);
}
#else
...
```
This will return a monotonic nanosecond timestamp like the unix version does
(i haven't tested this, but the internet tells me this should work)